neuraxle/base.py

```diff
@@ -29,14 +29,55 @@
 from neuraxle.hyperparams.space import HyperparameterSpace, HyperparameterSamples
 
 
-class BaseStep(ABC):
+class Hasher(ABC):
```

Comment:
to do : make 1 hasher to hash hyperparams

Reply:
1 or 2 (abstract)